action.yml

```diff
@@ -19,12 +19,71 @@ inputs:
       `output_format` parameter for accepted values.
     required: false
     default: "all"
+  no_cache:
+    description: "Disable github action R dependency caching"
+    required: false
+    default: "false"
+  cache_version:
+    description: "Version of the cache. To clean cache bump this version."
+    required: false
+    default: "cache-v1"
+branding: # https://feathericons.com/
+  icon: "award"
+  color: "blue"
 
-# Run as a Docker action
 runs:
-  using: 'docker'
-  image: './Dockerfile'
+  using: "composite"
+  steps:
+    - name: Get R version
+      id: r_version
+      run: echo "::set-output name=R_VERSION::$(R --version | head -1 | awk '{print $3}')"
+      shell: bash
 
-branding: # https://feathericons.com/
-  icon: 'award'
-  color: 'blue'
+    - name: Set R Library home on Linux
+      
+      run: |
+        mkdir -p /__w/_temp/Library
+        echo ".libPaths(c('/__w/_temp/Library', .libPaths()))" > ~/.Rprofile
+      shell: bash
+
+    - name: Set tlmgr cache folder
+      if: "contains(inputs.no_cache, 'false')"
+      id: texlive_version
+      run: |
+        echo "TEXMFHOME=/__w/_temp/texlive/texmf-home" >> $GITHUB_ENV
+        echo "TEXMFVAR=/__w/_temp/texlive/texmf-var" >> $GITHUB_ENV
+        export TEXMFHOME=/__w/_temp/texlive/texmf-home
+        export TEXMFVAR=/__w/_temp/texlive/texmf-var
+        tlmgr init-usertree
+        echo "::set-output name=TEX_LIVE_VERSION::$(tlmgr --version |tail -1 |awk '{print $NF}')"
+      shell: bash
```

Comment:
Might be good to install TinyTex instead of relying on a TexLive installation.

Reply:
done